src/GafferOSL/OSLObject.cpp

```diff
@@ -151,6 +166,8 @@ void OSLObject::hashProcessedObject( const ScenePath &path, const Gaffer::Contex
 	{
 		shader->attributesHash( h );
 	}
+
+	interpolationPlug()->hash( h );
```

Comment:
Thanks for squashing that middle commit down. One note for the future - getting affects/hash/compute to match is one of the most important things in coding a node, so it's best not to split the hash/compute changes across commits like this as it makes review hard. I'd have been quite happy to see this commit squashed into the next too...